src/main/java/uk/gov/hmcts/reform/idam/health/vault/KeyVaultConfig.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.idam.health.vault;
+
+import org.springframework.core.env.ConfigurableEnvironment;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
```

Comment:
Do we actually need setters in this class?

Reply:
Nope, will remove.